lib/types.ts

```diff
@@ -3,6 +3,16 @@ import type {Merge} from 'type-fest';
 import type {BuiltIns} from 'type-fest/source/internal';
 import type OnyxUtils from './OnyxUtils';
 
+/**
+ * Utility type that excludes `null` from the type `TValue`.
+ */
+type NonNull<TValue> = TValue extends null ? never : TValue;
+
+/**
+ * Utility type that excludes `undefined` from the type `TValue`.
+ */
+type NonUndefined<TValue> = TValue extends undefined ? never : TValue;
+
```

Comment:
Would [NonNullable](https://www.typescriptlang.org/docs/handbook/utility-types.html#nonnullabletype) work here?

Reply:
`NonNullable` excludes both `undefined` and `null`, which is not what I want here 😕